src/hashtable.py

```diff
@@ -63,7 +71,16 @@ def remove(self, key):
 
         Fill this in.
         '''
-        pass
+        index = self._hash_mod(key)
+        node = self.storage[index]
+        while node is not None:
+            if key == node.key:
```

Comment:
I feel like it would be cleaner if you removed the node entirely here, but I understand how this could work in general